server/boot/user.js

```diff
@@ -249,7 +249,9 @@ module.exports = function(app) {
   }
 
   function getSettings(req, res) {
-    res.render('account/settings');
+    res.render('account/settings', {
+        title: 'Settings | Free Code Camp'
```

Comment:
The changed one will show:

![2](https://cloud.githubusercontent.com/assets/1884376/13845252/ba82b44e-ec64-11e5-9ced-ac02ab35b136.png)